docker-compose.yml

```diff
@@ -59,3 +58,15 @@ services:
 #      HTTPS config - uncomment to enable redirect from :80 to :443
 #      - "traefik.http.routers.${COMPOSE_PROJECT_NAME}.middlewares=redirect-to-https"
 #      - "traefik.http.middlewares.redirect-to-https.redirectscheme.scheme=https"
+
+  rabbit:
+    image: rabbitmq:3.9-management
+    networks:
+      - app
+      - frontend
+    ports:
+      - "15672"
+    environment:
+      - RABBITMQ_DEFAULT_USER=user
+      - RABBITMQ_DEFAULT_PASS=password
+      - RABBITMQ_ERLANG_COOKIE='d53f219cd9376f8f440aaf9889f315ab'
```

Comment:
Should this be moved to `docker-compose.override.yml` to keep `docker-compose.yml` clean and identical to the template file (https://github.com/itk-dev/devops_itkdev-docker/blob/develop/templates/symfony-6/docker-compose.yml)?

This is probably a more general questions and not directly related to this pull request.

Reply:
I see your point but will refer it to a general discussion about how best to structure our docker compose files.